packages/app/src/cli/services/dev/processes/draftable-extension.ts

```diff
@@ -128,6 +128,8 @@ export async function setupDraftableExtensionsProcess({
     envIdentifiers: prodEnvIdentifiers,
   })
 
+  localApp.updateExtensionUUIDS(extensionsUuids)
```

Comment:
maybe a comment here to explain why the temp devUuids, generated each time the `dev` command is run, should be overwritten with the remote ones (in case they exist)